docs/tutorials/microbit-basic.md

```diff
@@ -0,0 +1,107 @@
+
+Connect the jumper wires between the following pins:
+
+micro:bit   | EE-NBIOT-01
+-----------:|------------
+3V - red    | VCC
+GND - black | GND
+P0 - yellow | RXD
+P1 - orange | TXD
+
+### Edge connector with pin headers
+The recommended method to connect a micro:bit to EE-NBIOT-01
+<a href="img/microbit-edge-connector.jpg" target="_blank">![Using an edge connector with pin headers](img/microbit-edge-connector-small.jpg)<br/>
+See image in full resolution</a>
+
+
+### Improvise wire connection
+If you don't have an edge connector nearby and don't have time to wait for one, check out [this guide](https://lorrainbow.wordpress.com/2017/11/11/how-to-connect-wires-to-the-microbit/). Or improvise using what you have available.
+![Improvise connection](img/microbit-screws-small.jpg)
+
+## Register device
+Before we start prototyping, we need the IMEI and IMSI and register the device in our NB-IoT Developer Platform:
+
+1. Open [this makecode project](https://makecode.microbit.org/_gfV43eEt51Er) and click the «Edit»-button
+1. Use a micro-USB cable to connect the micro:bit to your computer
+1. The micro:bit should show up as a regular USB memory stick on your computer, called MICROBIT
+1. Click the ![big download](img/makecode-download.png) button to download the sketch and save it to the MICROBIT drive
+1. The sketch will now magically be flashed to the micro:bit
+1. After a few seconds you should see «PRESS A/B» scroll by on the LEDs on the micro:bit
+1. Get a pen and paper, press the A/B buttons and write down the numbers
+1. Follow our [Getting started with the Telenor NB-IoT Developer Platform guide](getting-started.html) and use the IMEI/IMSI from the step above
+1. Leave the browser tab with live device data open, so we can see data we send in the next section
+
+## Install nbiot extension
+1. Go to [makecode.microbit.org](https://makecode.microbit.org/)
+1. Click on the big *«New Project»-button*
+1. Click the *gear* in the top right, then *Extensions*
+    - ![Extensions screenshot](img/makecode-extensions.png)
+1. In the search field, enter the URL below and click the search button
+    1. https://github.com/ExploratoryEngineering/pxt-nbiot
+    1. Select «nbiot» in the search result
+    - ![nbiot extension](img/makecode-nbiot-install.png)
+1. You should now get an «NB-IoT» category in the blocks menu
+    - ![nbiot menu](img/makecode-nbiot-menu.png)
+
+
+## Hello World
+Drag-n-drop blocks from the menu to reproduce the blocks below:
+
+![Hello World example](img/microbit-hello.png)
+
+### What's going on here?
+In the block editor, every chain of blocks starts with an event (something happened). All the blocks inside of it will be executed in order from top to bottom.
```

Comment:
The (something happened) part makes perfect sense to me, but perhaps you could elaborate a bit one it? Like. what's the alternative?

Reply:
What _is_ the alternative? Please elaborate 🙂